discv5-utp.md

```diff
@@ -76,8 +76,8 @@ A typically flow of messages:
         Bob->>Alice: uTP ST_DATA
         Alice->>Bob: ...
         Bob->>Alice: ...
-        Note left of Bob: Once DATA send & acknowledged
-        Bob->>Alice: uTP ST_FIN
```

Comment:
Ok, I see. In that case I suggest annotating that in the diagram, as the sentence below the diagram appears to describe the ST_FIN message.

Reply:
Yes, the typical flow would be that Bob sends the ST_FIN. But Alice MAY also send a ST_FIN, and there are situations where this may happen (e.g. lost ST_FIN packet).

Feel free to reorder text/annotations to make that more clear.